.github/workflows/docker-frontend.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - id: tag
         run: |
           echo "GITHUB_REF: $GITHUB_REF"
-          TAG=$(echo ${GITHUB_REF} | sed -e 's/refs\/heads\/master/latest/g' | sed -e 's/refs\/tags\///g')
+          TAG=$(echo ${GITHUB_REF} | sed -e 's,refs/heads/master,latest,g' | sed -e 's,refs/tags/,,g' | sed -e 's,refs/pull/\([0-9]*\).*,pr\1,g')
```

Comment:
At this point I'm a bit torn between combining these into a giant regex or abusing command piping :P

Reply:
They're all different replacements, so there is no one regex, I don't think. I can just remove the pipes and have multiple `-e` flags, if you want.

Reply:
Sure that would look better. Pipes add confusion here.